include/engine/plugins/trip.hpp

```diff
@@ -145,7 +137,7 @@ template <class DataFacadeT> class RoundTripPlugin final : public BasePlugin
             // whole graph is one scc
             std::vector<NodeID> location_ids(number_of_locations);
             std::iota(std::begin(location_ids), std::end(location_ids), 0);
-            return SCC_Component(std::move(location_ids));
+            return SCC_Component(std::move(location_ids), std::vector<size_t> {{0, location_ids.size()}});
```

Comment:
Wait, now the problem is a bit different, isn't it? What we are now running into is the difference between

    vector<int> x(1, 42); // (count, value) ctor - vector of one initalized int with value 42

and

    vector<int> x{1, 42}; // vector of two ints with value 1 and 42

FYI, small trick to make things like this easier to test: on freenode there are bots - `geordi` and `clang` - that let you quickly test those small snippets, e.g.

    { vector<size_t> x = {0, 42}; cout << x; }
    <geordi> {0, 42}

Just in case you're on freenode anyway.

Reference:
- http://en.cppreference.com/w/cpp/container/vector/vector